diff --git a/dist/utils/json-parse.js b/dist/utils/json-parse.js
index 82e105cb676a2f4be16167023243aa73dd8cb2e4..fdddc7b9b6bd96d7bac17ab9eb42a7b0beeb7465 100644
--- a/dist/utils/json-parse.js
+++ b/dist/utils/json-parse.js
@@ -1,4 +1,4 @@
-import { parse as partialParse } from "partial-json";
+import { partialParse } from "openai/_vendor/partial-json-parser/parser.mjs";
 const VALID_JSON_ESCAPES = new Set(['"', "\\", "/", "b", "f", "n", "r", "t", "u"]);
 function isControlCharacter(char) {
     const codePoint = char.codePointAt(0);
